project/ws/app/src/lib/routes/home/routes/odcs-mapping/bulk-upload-odcs/bulk-upload-odcs.component.html

```diff
@@ -0,0 +1,109 @@
+<div class="flex flex-col gap-6 mt-4">
+  <div class="flex flex-row gap-4 items-center">
+    <mat-icon class="cursor-pointer" [routerLink]="['/app/home/odcs-mapping']">arrow_back</mat-icon>
+    <div class="flex font-bold">Bulk Upload Competency Designation</div>
+  </div>
+
+  <div class="flex flex-col gap-4 bulk-container">
+    <div class="flex font-bold">Bulk Upload Competency Designation</div>
+    <div class="flex flex-row gap-8">
+      <div class="flex-1">
+        <div class="flex flex-col gap-3 dashed-card">
+          <div>Follow these steps to update multiple user profiles:</div>
+          <div class="flex flex-col gap-4">
+            <ol class="custom-ol">
+              <li>Download the Sample File ZIP.</li>
+              <li>Fill in user details in the Sample File.</li>
+              <li>Verify via OTP.</li>
+              <li>Upload the populated sample csv file.</li>
+              <li>Check the upload results for confirmation.</li>
+            </ol>
+          </div>
+          <div class="flex flex-row gap-6">
+            <button class="outline-button" (click)="handleDownloadSampleFile()">Download Sample File</button>
+          </div>
+        </div>
+      </div>
+      <div class="flex-1">
+        <div class="flex flex-col gap-2 dashed-card items-center">
+          <div class="flex cursor-pointer margin-top-m" (click)="fileInput.click()"
+            (dblclick)="$event.preventDefault()">
+            <img src="/assets/icons/csv-file.svg" alt="CSV file logo" />
+          </div>
+          <div *ngIf="showFileError" class="error-msg">Upload xlsx format file only</div>
+          <div class="flex flex-col items-center">
+            <div class="text-primary cursor-pointer" (click)="fileInput.click()" (dblclick)="$event.preventDefault()">
+              Browse files</div>
+            <input hidden type="file" #fileInput (click)="handleFileClick($event)" (change)="handleOnFileChange($event)"
+              name="file" placeholder="Choose file" (dblclick)="$event.preventDefault();" accept=".xlsx, .xls" />
+            <!-- <div class="flex">or</div>
+            <div class="flex">Drag file to upload</div> -->
+          </div>
+          <div class="flex flex-col items-center margin-bottom-m">
+            <div class="flex">Max file size: 100 MB</div>
```

Comment:
all strings to be moved to json in the file

Reply:
Added in config